pkgs/tools/networking/dnsmasq/default.nix

```diff
@@ -1,13 +1,17 @@
-{ lib, stdenv, fetchurl, pkg-config, dbus, nettle, fetchpatch
-, libidn, libnetfilter_conntrack, buildPackages }:
+{ lib, stdenv, fetchurl, pkg-config, nettle, fetchpatch
+, libidn, libnetfilter_conntrack, buildPackages
+, dbusSupport ? true
```

Comment:
To have the exact some behaviour, this should default to `stdenv.isLinux`.

Reply:
Agreed, thank you.